.github/workflows/build_lint_test.yml

```
@@ -2,9 +2,9 @@ name: "Python package: build, lint, and test"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "tests" ]
```

```suggestion
    branches: [ "main" ]
```

Suggest we default to only running automatically on the `main` branch to avoid excessive use of GitHub Actions minutes.

The `on.workflow_dispatch:` key allows the workflow to be manually run on any branch from the Actions tab on GitHub, e.g. for build, lint, test workflow, go to <https://github.com/isambard-sc/bricsauthenticator/actions/workflows/build_lint_test.yml> then select the "Run workflow" drop-down menu.